src/core/main.js

```diff
@@ -729,6 +729,36 @@ p5.instance = null;
  */
 p5.disableFriendlyErrors = false;
 
+/**
+ * Normally, p5 will try to create a WebGL2 context when using either
+ * `createCanvas(width, height, WEBGL)` or `createCanvas(width, height, WEBGL)`,
+ * and will fall back to a WebGL1 context if WebGL2 is not available. If you
+ * want to always use a WebGL1 context, you can set `p5.disableWebGL2 = true`.
+ *
+ * @property {Boolean} disableWebGL2
```

Comment:
could we do this in the setAttributes function instead? It feels against the grain of existing p5 patterns to be setting something on the p5 object before the sketch is created. Something like:
```
setAttributes('version' 1);
// or
setAttributes({version: 2});
```
Is there a reason we wouldn't do it this way?